### PR TITLE
br: add doc for latest backup protection

### PR DESCRIPTION
**Pull request by grovecai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/backup-and-restore.md [hunk 1]
```diff
@@ -91,8 +91,9 @@ To configure the backup schedule, perform the following steps:
 
         > **Note**
         >
-        > - After you delete a cluster, the automatic backup files will be retained for a specified period, as set in backup retention. You need to delete the backup files accordingly.
-        > - After you delete a cluster, the existing manual backup files will be retained until you manually delete them, or your account is closed.
+        > - Auto-backups except the latest auto-backup will be deleted if the lifetime exceed the retention period. For the latest auto-backup, it will not be deleted until you delete it explicitly. This will prevent the cluster is deleted mistakenly and no data to restore.
+        > - After you delete a cluster, auto-backup which lifetime is not exceed the retention period will be retained in Recycle Bin. Auto-backups except the latest auto-backup will be deleted if the lifetime exceed the retention period. For the latest auto-backup, it will not be deleted until you delete it explicitly. This will prevent the cluster is deleted mistakenly and no data to restore.
+        > - After you delete a cluster, the existing manual backups will be retained until you manually delete them, or your account is closed.
```
Comment (hfxsd) [inline tidb-cloud/backup-and-restore.md:96]:
```suggestion
        > - All auto-backups, except the latest one, will be deleted if their lifetime exceeds the retention period. The latest auto-backup will not be deleted unless you delete it explicitly. This ensures that the cluster is not accidentally deleted without the ability to restore data.
        > - After you delete a cluster, auto-backups with a lifetime within the retention period will be moved to the Recycle Bin. Auto-backups, except the latest one, will be deleted if their lifetime exceeds the retention period. The latest auto-backup will not be deleted unless you delete it explicitly. This prevents accidental cluster deletion with no data to restore.
        > - After you delete a cluster, existing manual backups will be retained until manually deleted or your account is closed.
```

File: tidb-cloud/backup-and-restore.md [hunk 2]
```diff
@@ -91,8 +91,9 @@ To configure the backup schedule, perform the following steps:
 
         > **Note**
         >
-        > - After you delete a cluster, the automatic backup files will be retained for a specified period, as set in backup retention. You need to delete the backup files accordingly.
-        > - After you delete a cluster, the existing manual backup files will be retained until you manually delete them, or your account is closed.
+        > - All auto-backups, except the latest one, will be deleted if their lifetime exceeds the retention period. The latest auto-backup will not be deleted unless you delete it explicitly. This ensures that the cluster is not accidentally deleted without the ability to restore data.
+        > - After you delete a cluster, auto-backups with a lifetime within the retention period will be moved to the Recycle Bin. Auto-backups, except the latest one, will be deleted if their lifetime exceeds the retention period. The latest auto-backup will not be deleted unless you delete it explicitly. This prevents accidental cluster deletion with no data to restore.
```
Comment (hfxsd) [inline tidb-cloud/backup-and-restore.md:95]:
```suggestion
        > - After you delete a cluster, auto-backups with a lifetime within the retention period will be moved to the Recycle Bin. 
```

File: tidb-cloud/backup-and-restore.md [hunk 3]
```diff
@@ -91,8 +91,9 @@ To configure the backup schedule, perform the following steps:
 
         > **Note**
         >
-        > - After you delete a cluster, the automatic backup files will be retained for a specified period, as set in backup retention. You need to delete the backup files accordingly.
-        > - After you delete a cluster, the existing manual backup files will be retained until you manually delete them, or your account is closed.
+        > - All auto-backups, except the latest one, will be deleted if their lifetime exceeds the retention period. The latest auto-backup will not be deleted unless you delete it explicitly. This ensures that the cluster is not accidentally deleted without the ability to restore data.
```
Comment (hfxsd) [inline tidb-cloud/backup-and-restore.md:94]:
```suggestion
        > - All auto-backups, except the latest one, will be deleted if their lifetime exceeds the retention period. The latest auto-backup will be retained unless you delete it explicitly. This ensures that the cluster is not accidentally deleted without the ability to restore data.
```

Comment (hfxsd) [inline tidb-cloud/backup-and-restore.md:94]:
```suggestion
        > - All auto-backups, except the latest one, will be deleted if their lifetime exceeds the retention period. The latest auto-backup will not be deleted unless you delete it explicitly. This ensures that a cluster still has data to restore after it is accidentally deleted.
```

File: tidb-cloud/backup-and-restore.md [hunk 4]
```diff
@@ -91,8 +91,9 @@ To configure the backup schedule, perform the following steps:
 
         > **Note**
         >
-        > - After you delete a cluster, the automatic backup files will be retained for a specified period, as set in backup retention. You need to delete the backup files accordingly.
-        > - After you delete a cluster, the existing manual backup files will be retained until you manually delete them, or your account is closed.
+        > - All auto-backups, except the latest one, will be deleted if their lifetime exceeds the retention period. The latest auto-backup will not be deleted unless you delete it explicitly. This ensures that a cluster still has data to restore after it is accidentally deleted.
+        > - After you delete a cluster, auto-backups with a lifetime within the retention period will be moved to the Recycle Bin.
```
Comment (Oreoxmt) [inline tidb-cloud/backup-and-restore.md:95]:
```suggestion
        > - After you delete a cluster, auto-backups with a lifetime within the retention period will be moved to the recycle bin.
```
Keep the capitalization consistent with:
https://github.com/pingcap/docs/blob/a0100a3605f5e062367e1d8d3b3017c412986429/tidb-cloud/backup-and-restore.md?plain=1#L9

File: tidb-cloud/backup-and-restore.md [hunk 5]
```diff
@@ -91,8 +91,9 @@ To configure the backup schedule, perform the following steps:
 
         > **Note**
         >
-        > - After you delete a cluster, the automatic backup files will be retained for a specified period, as set in backup retention. You need to delete the backup files accordingly.
-        > - After you delete a cluster, the existing manual backup files will be retained until you manually delete them, or your account is closed.
+        > - All auto-backups, except the latest one, will be deleted if their lifetime exceeds the retention period. The latest auto-backup will not be deleted unless you delete it explicitly. This ensures that a cluster still has data to restore after it is accidentally deleted.
```
Comment (Oreoxmt) [inline tidb-cloud/backup-and-restore.md:94]:
```suggestion
        > - All auto-backups, except the latest one, will be deleted if their lifetime exceeds the retention period. The latest auto-backup will not be deleted unless you delete it manually. This ensures that you can restore cluster data if accidental deletion occurs.
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by hfxsd — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Pull request merged**